frontend/src/App.tsx

```diff
@@ -124,14 +127,34 @@ export function update(msg: Msg, model: Model): [Model, cmd.Cmd<Msg>] {
         }
         , findContractCmd(model, model.addressInput)
       ]
+
     case 'FindContractResult':
       console.log("Contract Info", msg.result)
       let contract = pipe(msg.result, E.fold(RD.failure, r => RD.success(r)))
 
+      let getStorageOnSucceed = () => {
+        if (contract._tag === "RemoteSuccess")
+          return getStorageCmd(model, contract.value)
+        else return cmd.none
+      }
+
       return [
         {
           ...model,
-          contract
+          contract,
+          storage: RD.pending
+        }
+        , getStorageOnSucceed()
+      ]
+
+    case 'GetStorageResult':
+      console.log("what", msg.result)
+      let getStorageResult = pipe(msg.result, E.fold(RD.failure, r => RD.success(r)))
+
+      return [
+        {
+          ...model,
+          storage: getStorageResult
         }
         , cmd.none
       ]
```

Comment:
It seems that I'm unable to connect, I believe (from your question on Slack) that the hardcoded URL a few lines below should be `https://granada.testnet.tezos.serokell.team/`.

Also, I think both of these URL should be settable, I'd like to see them as options in the make target too.